### DISCUSSION TITLE
no info on character LCD while playing music from network/NAS

### DISCUSSION THREAD

**Discussion by gitso7** [rern/rAudio#765] [Fixed Issues]
Hi
I attached a 20x4 character LCD attached at I2C.
I see infos like artist, song and album on the LCD while playing one of the four webradio stations «Paradise...>.
I don't see anything (=blank LCD) while playing other webadio stations.
And most troubling, I see nothing but a blank LCD while playing a song from my NAS.
The web page raudio.local always tells me artist, song and album.
What's going wrong? What can I do?
Thanks for any help.
Peter

**Comment by gitso7** [comment]
in addition:
Is anyone using a (20x4 character) LCD that shows infos like artist, song and album on the LCD while playing songs from a network drive/NAS?

**Reply by rern** [reply]
Try `20220117` update.

**Comment by gitso7** [comment]
Hi rern

problem solved
artist, song and album are nicely shown on the LCD while playing from network/NAS and all the webradio stations I checked
thanks a  L O T
and I appreciate your work
